featherbed-core/src/main/scala/featherbed/request/RequestSyntax.scala

```diff
@@ -68,6 +70,77 @@ trait RequestTypes { self: Client =>
       canBuild: CanBuildRequest[Self]
     ): ValidatedNel[Throwable, Request] = canBuild.build(this: Self)
 
+    private def cloneRequest(in: Request) = {
+      val out = Request()
+      out.uri = in.uri
+      out.content = in.content
+      in.headerMap.foreach {
```

Comment:
Wouldn't cookies be included in the headers?

The alternative to cloning the request is mutating the URI in the original request.  I don't like the sound of that, but I suppose it wasn't passed in originally by the user so maybe it would be ok?